packages/angular/stories/scatter.stories.ts

```diff
@@ -0,0 +1,31 @@
+import { storiesOf } from "@storybook/angular";
+
+import { colors } from "./helpers/commons";
+
+import { ChartsModule } from "../src/charts.module";
+import { ScatterComponent } from "./scatter/scatter.component";
+
+import { scatterData, lineOptions } from "./../../core/demo/demo-data/line";
+
+const pieStories = storiesOf("Scatter", module);
+pieStories.add("Basic", () => ({
+	component: ScatterComponent,
+	moduleMetadata: {
+		imports: [ChartsModule]
+	},
+	props: {
+		scatterData: scatterData,
+		scatterOptions: lineOptions
+	}
+}));
+
+pieStories.add("Accessible", () => ({
```

Comment:
Technically scatter is not accessible yet

Reply:
Can we duplicate these stories for scatter in angular storybooks please?
https://github.com/carbon-design-system/carbon-charts/blob/master/packages/react/stories/line-step.stories.js

Reply:
I removed the accessible from the scatter story. I'm not proficient in React so would leave that to you or somebody else to do.